src/Features/CSharp/Portable/Structure/Providers/RegionDirectiveStructureProvider.cs

```diff
@@ -46,14 +47,9 @@ private static string GetBannerText(DirectiveTriviaSyntax simpleDirective)
                     textSpan: TextSpan.FromBounds(regionDirective.SpanStart, match.Span.End),
                     type: BlockTypes.PreprocessorRegion,
                     bannerText: GetBannerText(regionDirective),
-                    autoCollapse: autoCollapse,
-                    isDefaultCollapsed: true));
+                    autoCollapse: isMetadataAsSource || autoCollapse,
+                    isDefaultCollapsed: !isMetadataAsSource));
```

Comment:
ok.  so this applies because MAS emits some regions?  this confused me a while, might be worth a comment

Reply:
Added comments